demos/app/iot_demo.c

```diff
@@ -118,19 +118,55 @@ int main( int argc,
     /* For a secured connection, set the members of the credentials. */
     if( demoArguments.securedConnection == true )
     {
-        /* Set credential paths. */
+        /* Set credential information. */
         credentials.pClientCert = demoArguments.pClientCertPath;
         credentials.pPrivateKey = demoArguments.pPrivateKeyPath;
         credentials.pRootCa = demoArguments.pRootCaPath;
+        
```

Comment:
If you add the following here, lines 134-137 and 148-151 can be deleted.

```c
credentials.pUserName = NULL;
credentials.pPassword = NULL;
```

Reply:
Good catch, thank you, fix is pending.